src/Components/FormCreation/PublishedFormHeader.tsx

```diff
@@ -0,0 +1,218 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import Button from "@material-ui/core/Button";
+import Snackbar from "@material-ui/core/Snackbar";
+import PublishedFormDialog from "./Dialogs/PublishedFormDialog";
+import ManageApplicantAccessDialog from "./Dialogs/ManageApplicantAccessDialog";
+import CheckCircleOutlineIcon from "@material-ui/icons/CheckCircleOutline";
+import SettingsIcon from "@material-ui/icons/Settings";
+
+const DialogOverlay = styled.div`
+  position: fixed;
+  left: 0;
+  top: 0;
+  height: 150%;
+  width: 100vw;
+  z-index: 110;
+  background: rgba(0, 0, 0, 0.5);
+  .dialogButton {
+    text-transform: none;
+  }
+`;
+
+const Header = styled.div`
+  box-sizing: border-box;
+  height: 201px;
+  width: 100%;
+  box-shadow: 0 2px 3px 1px #cccccc;
+  display: flex;
+  background: #fafafa;
+`;
+
+const PublishedHeader = styled.div`
+  box-sizing: border-box;
+  height: 166px;
+  width: 100%;
+  background: #55a94e;
+  display: flex;
+`;
+
+const TextWrapper = styled.div`
+  padding-left: 15%;
+  padding-right: 0px;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+`;
+
+type Props = {
+  name: string;
+  description: string;
+  submissionLink: string;
+  onChange: (name: string, description: string) => void;
+};
+
+function PublishedFormHeader({
+  description,
+  submissionLink
+}: Props): React.ReactElement<typeof Header> {
+  const [showPublishedFormDialog, setshowPublishedFormDialog] = useState(false);
+  const [showManageAccessDialog, setshowManageAccessDialog] = useState(false);
+  const [copiedsubmissionLink, setCopiedsubmissionLink] = useState(false);
+
+  function closeDialog() {
+    setshowPublishedFormDialog(false);
+    setshowManageAccessDialog(false);
+  }
+
+  const copyLinkToClipboard = () => {
+    navigator.clipboard.writeText(submissionLink);
+    var dummy = document.createElement("input");
+    dummy.innerText = submissionLink;
+    document.body.appendChild(dummy);
+    dummy.setAttribute("id", "dummy_id");
+    dummy.select();
+    document.execCommand("copy");
+    dummy.remove();
+    setCopiedsubmissionLink(true);
+  };
+
+  return (
+    <div>
+      <PublishedHeader>
+        <TextWrapper>
+          <div
+            style={{
+              display: "flex",
+              alignItems: "center"
+            }}
+          >
+            <CheckCircleOutlineIcon
+              style={{ color: "white", marginRight: 20 }}
+            />
+            <p style={{ color: "white", fontSize: 20 }}>
+              {" "}
+              This form has been published to applicants.{" "}
+            </p>
+          </div>
+          <p style={{ color: "white" }}>
+            {" "}
+            If you published the form erroneously, make a copy of this form to
+            keep editing.{" "}
```

Comment:
Copy update: "If you published the form erroneously, make a copy of this program to keep editing."